01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,17 @@
+const http = require('http')
+
+const server = http.createServer((req, res) => {
+  if (req.url === '/') {
+    res.end('Welcome to our home page')
+  } else if (req.url === '/about') {
+    res.end('Here is our short history')
+  } else {
+    res.end(`
+    <h1>Oops!</h1>
+    <p>We can't seem to find the page you are looking for</p>
+    <a href="/">back home</a>
+    `)
+  }
+})
```

Comment:
(optional) extra challenge: can you add a page to this "website"

Reply:
added a contact page :) 

https://github.com/Code-the-Dream-School/node-express-course/pull/24/commits/bac04824524cf9c18d540e3d1fc6746d802a9cbf

Reply:
> (optional) extra challenge: can you add a page to this "website"
const server = http.createServer((req, res) => {
  if (req.url === '/') {
    res.end('Welcome to our home page')
  } else if (req.url === '/about') {
    res.end('Here is our short history')
  } else if (req.url === '/services') {
    res.end('All the services are well maintained!')
  } else {
    res.end(`
    <h1>Oops!</h1>
    <p>We can't seem to find the page you are looking for</p>
    <a href="/">back home</a>
    `)
  }
})

Reply:
Looks good, nice job!